### DISCUSSION TITLE
2 Monitors

### DISCUSSION THREAD

**Discussion by popi14** [awawa-dev/HyperHDR#982] [Q&A] [answered]
Is there any way i can make HyperHDR capture both monitors?

**Comment by awawa-dev** [comment]
yes

**Reply by popi14** [reply]
How can this be done? In settings I can only choose one monitor out of two.

**Reply by awawa-dev** [reply, marked as answer]
its a new feature https://github.com/awawa-dev/HyperHDR/pull/966
